api/src/main/java/org/apache/iceberg/view/ViewVersion.java

```diff
@@ -78,5 +78,10 @@ default String operation() {
   default void check() {
     Preconditions.checkArgument(
         summary().containsKey("operation"), "Invalid view version summary, missing operation");
+    String operation = summary().get("operation");
+    Preconditions.checkArgument(
+        "create".equals(operation) || "replace".equals(operation),
+        "Invalid operation. Must be create or replace but was: %s",
+        operation);
```

Comment:
Won't this prevent loading any view with a bad operation? That doesn't seem like something that should completely break a view.